### DISCUSSION TITLE
Versions with react@15.5.4

### DISCUSSION THREAD

**Discussion by 0jimah** [speechly/speechly#268] [General]
I am writing about the version compatibility of @speechly/react-client, is there any version that is compatible with react version 15.5.4?. Currently trying it with 2.2.1 and it's not working.

**Comment by thisismatu** [comment]
Hi, `@speechly/react-client` is compatible with React versions 16-18. You could try an earlier version (maybe 2.1.0 or earlier) at your own risk. Using an older version might introduce other anomalies. React 15 is quite out of date, why not use this as an excuse to upgrade it? :)
